force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -191,23 +285,36 @@ public with sharing class VariablesDataTypesOperators {
      */
     public static Integer adjustOrderOfOperations1(Integer a, Integer b, Integer c) {
         // Add parentheses around the addition operation so that it is performed before multiplication
-        Integer result = a + b * c;
-
-        // Return the result
-        return null; // Replace null with the variable you used to store the result
-    }
+        
+        If ((a == null) || (b == null) || (c == null))
+        {
+            return null;
+        }
+        Integer result = (a + b) * c;
+                
+        if (result == null)
+        {
+            return null;
+        }
+        return result;
+        
+       }
 
     /**
      * Question 14
      * Modifies the expression to adhere to PEMDAS rules in order to change its result.
      * @return The result of the expression after modifications.
      */
     public static Integer adjustOrderOfOperations2() {
-        // Add parentheses in the below expression to change the result.
-        // The result of the expression as it is right now is 43. 
-        // You should add parentheses so that the result of the expression becomes 8.
-        Integer answer = 48 - 15 + 5 * 2;
-        return null; // Replace null with the variable you used to store the result
+       
+        Integer answer = 48 - ((15 + 5) * 2);
+        If(answer == null)
```

Comment:
It's good that you're aware of null but it would be almost impossible to get a null answer, since the values are provided to you and none of them are actually null